tests/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Joe Drago. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+# With testing enabled, all targets referenced by add_test() can be run
+# at once with CMake's ctest command line tool from the build folder.
+enable_testing()
+# Using a CMake FIXTURES_SETUP/FIXTURES_CLEANUP to create and delete a
+# subdirectory would be cleaner but this is way simpler.
+set(AVIF_TEST_TMP_DIR ${PROJECT_BINARY_DIR})
+
+add_executable(aviftest aviftest.c)
+if(AVIF_LOCAL_LIBGAV1)
+    set_target_properties(aviftest PROPERTIES LINKER_LANGUAGE "CXX")
+endif()
+target_link_libraries(aviftest avif ${AVIF_PLATFORM_LIBRARIES})
+
+add_executable(avifgridapitest avifgridapitest.c)
+if(AVIF_LOCAL_LIBGAV1)
+    set_target_properties(avifgridapitest PROPERTIES LINKER_LANGUAGE "CXX")
+endif()
+target_link_libraries(avifgridapitest avif ${AVIF_PLATFORM_LIBRARIES})
+add_test(NAME avifgridapitest COMMAND avifgridapitest)
```

Comment:
In this original code, this `add_test()` call is:

```
    add_test(NAME avifgridapitest
        COMMAND ${CMAKE_BINARY_DIR}/avifgridapitest)
```

So you removed `${CMAKE_BINARY_DIR}/` from the COMMAND. Is it because the test executable is now `${CMAKE_BINARY_DIR}/tests/avifgridapitest`?

Reply:
I think it's because
https://cmake.org/cmake/help/latest/command/add_test.html
> COMMAND
    Specify the test command-line. If <command> specifies an executable target (created by [add_executable()](https://cmake.org/cmake/help/latest/command/add_executable.html#command:add_executable)) it will automatically be replaced by the location of the executable created at build time.

Reply:
Thank you for the answer, Christopher.